script.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+let total = 0;
+let highest = 0;
+let stuAvg= 0;  
+var names= [];
+var grade= [];
+var pos1 = [];
+var arr = [];
+
+for (var i = 0; i < 10; i++) {             
+    names.push(prompt('Enter The Ten Student Names | Student: ' + (i + 1))); 
+    document.getElementById("stuNames").innerHTML=("Student's Names:" +names.join(','));

```

Comment:
You may want to add spacing here so that when the string is rendered, it's not all pushed together like `Student's Names:Derrick,Matt`
```suggestion
    document.getElementById("stuNames").innerHTML=("Student's Names: " + names.join(', '));

```